pkg/config/config.go

```diff
@@ -194,12 +206,11 @@ func InitCliConfig(configAndStacksInfo schema.ConfigAndStacksInfo, processStacks
 	}
 
 	if !configFound {
-		// If `atmos.yaml` not found, use the default config
-		// Set `ATMOS_LOGS_LEVEL` ENV var to "Debug" to see the message about Atmos using the default CLI config
+		// Use default config if no config was found in any location
 		logsLevelEnvVar := os.Getenv("ATMOS_LOGS_LEVEL")
 		if logsLevelEnvVar == u.LogLevelDebug || logsLevelEnvVar == u.LogLevelTrace {
-			u.PrintMessageInColor("'atmos.yaml' CLI config was not found in any of the searched paths: system dir, home dir, current dir, ENV vars.\n"+
-				"Refer to https://atmos.tools/cli/configuration for details on how to configure 'atmos.yaml'.\n"+
+			u.PrintMessageInColor("'atmos.yaml' CLI config was not found.\n"+
+				"Refer to https://atmos.tools/cli/configuration\n"+
 				"Using the default CLI config:\n\n", color.New(color.FgCyan))
```

Comment:
If we only emit this on Debug or Trace, seems like this should be emitted from a logging function.